### PR TITLE
Build ByteCode by make instead of shellExec

### PR DESCRIPTION
**Pull request by ladisgin**

### CODE REVIEW

File: server/src/KleeGenerator.cpp [hunk 1]
```diff
@@ -177,7 +177,18 @@ Result<fs::path> KleeGenerator::defaultBuild(const fs::path &hintPath,
     auto &command = optionalCommand.value();
     command.setSourcePath(sourceFilePath);
     command.setOutput(bitcodeFilePath);
-    auto [out, status, _] = ShellExecTask::executeUtbotCommand(command, buildDirPath, projectContext.projectName);
+
+    printer::DefaultMakefilePrinter makefilePrinter;
+    std::vector<fs::path> outfilePaths;
```
Comment (operasfantom) [inline server/src/KleeGenerator.cpp:182]: unused variable

File: server/src/KleeGenerator.cpp [hunk 2]
```diff
@@ -177,7 +177,18 @@ Result<fs::path> KleeGenerator::defaultBuild(const fs::path &hintPath,
...
+
+    fs::path output = command.getOutput();
```
Comment (operasfantom) [inline server/src/KleeGenerator.cpp:184]: unused variable

**Review by operasfantom — COMMENTED** [review]
Still can't figure out how these changes solve the main problem. Maybe you should have added a little explanation of what is going on underneath.

File: server/src/KleeGenerator.cpp (hunk 1 above)
Comment (ladisgin) [reply server/src/KleeGenerator.cpp]: deleted

File: server/src/KleeGenerator.cpp (hunk 2 above)
Comment (ladisgin) [reply server/src/KleeGenerator.cpp]: deleted

**Comment by ladisgin** [comment]
We use shell exec for build bytecode and pass argument to it by string.
For example compile command
`clang -DEXPORT= main.c` here EXPORT empty
it parse to exec: `clang`
args: `-DEXPORT=""`, `main.c`
when we pass this arguments shell exec real command will be like
`"clang" "-DEXPORT=\"\"" "main.c"` -> `clang -DEXPORT=\"\" main.c` here EXPORT empty string

**Comment by operasfantom** [comment]
> when we pass this arguments shell exec real command will be like `"clang" "-DEXPORT=\"\"" "main.c"` -> `clang -DEXPORT=\"\" main.c` here EXPORT empty string

Now the reason is clear for me, thanks. Will approve PR soon.

**Review by operasfantom — APPROVED** [review]

**Pull request merged**